Service/Runner.php

```diff
@@ -221,7 +224,7 @@ public function runRow(JobDefinition $jobDefinition, $action, $mode, $jobId)
 
         $temp = new Temp("docker");
         $temp->initRunFolder();
-        $workingDirectory = new WorkingDirectory($temp->getTmpFolder(), $this->loggersService->getLog());
+        $workingDirectory = new WorkingDirectory($temp->getTmpFolder(), $this->loggersService->getLog(), $component->getFeatures());
```

Comment:
vykaslal bych se na predavani features a radsi udelal Component::runAsRoot() a na to se ptal
to hasFeature je sice genericky, ale uz ted je ta jedna ficura rozplizla na dvou mistech a bude se blbe odstranovat